### PR TITLE
SQ Plugin: Remove deprecated import of integration test coverage from plugins

### LINKED ISSUE #4685: SQ Plugin: Remove deprecated import of integration test coverage from plugins
We should remove them because they have been deprecated since SonarQube 6.2!

```
    if (this.isIntegrationTest) {
      LOG.warn("Starting with SonarQube 6.2 separation between Unit Tests and Integration Tests Coverage" +
        " reports is deprecated. Please move all reports specified from *.it.reportPaths into *.reportPaths.");
    }
```

Repro steps:
1. Open a project in SonarCloud/SonarQube
2. Go to Administration -> General Settings -> Languages
3. Choose C#

It shows for example
image

However coverage for integration tests is deprecated.

The same, it shows NCover3
image

### PR DESCRIPTION
**Pull request by mary-georgiou-sonarsource**
Fixes #4685

### CODE REVIEW

**Comment by mary-georgiou-sonarsource** [comment]
I also checked S4NET for any appearance of `*.it.reportPaths` in the code or tests (UTs + ITs)

**Review by zsolt-kolbay-sonarsource — APPROVED** [review]
LGTM

File: sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/UnitTestResultsProviderTest.java [hunk 1]
```diff
@@ -43,8 +43,7 @@ public void vbnet() {
       UnitTestResultsProvider.DotNetUnitTestResultsAggregator.class,
       UnitTestResultsImportSensor.class);
     assertThat(propertyKeys(extensions)).containsOnly(
-      "sonar.vbnet.vstest.reportsPaths",
-      "sonar.vbnet.nunit.reportsPaths");
+      "sonar.vbnet.vstest.reportsPaths", "sonar.vbnet.nunit.reportsPaths");
```
Comment (andrei-epure-sonarsource) [inline sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/UnitTestResultsProviderTest.java:46]: @mary-georgiou-sonarsource do you find this to be more readable? personally I find separate lines to be more readable.

Comment (mary-georgiou-sonarsource) [reply sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/UnitTestResultsProviderTest.java:46]: Exactly. That was my thinking—that there are only two values, and separating them into two lines is not really needed—it does not add much apart from an extra line.
However, if you think it's better to separate not only for readability but also for uniformity with other similar cases, I'll be happy to revert in one of the next PRs.

Comment (andrei-epure-sonarsource) [reply sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/UnitTestResultsProviderTest.java:46]: I don't think we should spend too much time on discussing such topics (very low-impact)

**Pull request merged**